tests/config/BUILD

```diff
@@ -108,6 +108,19 @@ docker_toolchain_autoconfig(
     test = True,
 )
 
+docker_toolchain_autoconfig(
```

Comment:
Can we add the test for this to presubmit?

Reply:
done